node/lib/util/merge_util.js

```diff
@@ -129,294 +141,522 @@ exports.fastForwardMerge = co.wrap(function *(repo, commit, message) {
 });
 
 /**
- * Merge the specified `subs` in the specified `repo` having the specified
- * `index`.  Stage submodule commits in `metaRepo`.  Return an object
- * describing any commits that were generated and conflicted commits.  Use the
- * specified `opener` to acces submodule repos.  Use the specified `message` to
- * write commit messages.
- * @param {NodeGit.Repository} metaRepo
- * @param {Open.Opener}        opener
- * @param {NodeGit.Index}      metaIndex
- * @param {Object}             subs        map from name to SubmoduleChange
+ * Get the left side commit and sha for submodule `subRepo` based on workding
```

Comment:
spelling of working

Reply:
but also "based on working directory type" doesn't really make a lot of sense to me.

Reply:
actually, after revisiting the compute changes logic, the left side commit for submodules have already computed correctly in `CherryPickUtil.computeChangesBetweenTwoCommits`, so let me remove this